dist/css/webmakerLogin.css

```diff
@@ -401,7 +401,14 @@
   background: #32b2d2;
 }
 .modal-content .modal-body .checkbox input[type="checkbox"] {
-  display: none;
+  position: absolute;
+  left: -9000px;
+}
+.modal-content .modal-body .checkbox input[type="checkbox"]:focus + label div {
+  border: 2px solid #E1EBF1;
+  -webkit-box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.67);
```

Comment:
I don't think we need to prefix this, since caniuse.com says that it's been [supported for awhile](http://caniuse.com/#search=box-shadow).